Hcal/testbeam04-2022/pedestals/hcal_detids_hex.txt

```diff
@@ -0,0 +1,385 @@
+DetID
```

Comment:
This file may be interesting to keep around. We could use it to double check any tables being added/changed in the future.